vm-to-vm-bandwidth-meter/azuredeploy.json

```diff
@@ -0,0 +1,611 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "probeVmSize": {
```

Comment:
Can we make this a variable with a default that is globally available?  Otherwise I think we'll see too many failures trying to match region with VM Size availability...  and we don't have a maintenance task keeping this up to date.  The readme can underscored the role size plays here...